test/test_validation.jl

```diff
@@ -144,7 +144,7 @@ end
         (:error, r"Invalid range"),
         match_mode = :any,
         @test_throws(
```

Comment:
I think this test needs update from invalid range to value too. Now it passes because of the match  condition

Reply:
It's actually correct. We may need to discuss. I didn't realize this part when we merged the IS PR.